TRIDENT-15830 Scheduled Actions: Enhance Exit Condition Configuration (#3238)
SumoLogic/sumologic-documentation: docs/cloud-soar/cloud-soar-integration-framework.md

* **image_base64_png(jpg)* ** [String]: Result path of a base64 image png or jpg format.
* **signature** [String]: Signature to indicate action is the original one written by Sumo Logic.
* **exit_condition**: Specify what condition system has to evaluate to decide if continue with next execution or to stop scheduled action and continue with playbook next actions.
   * **path* ** [String]: Result path where to search in JSON structure as `table_view` section or specify an action's input i.e. input.path.

>    * **string* ** [String\List[String]]: Result path of string to check if is equal to result value or an array of string or specify an action's input i.e. input.matchString. 

Reviewer: ```suggestion
   * **string* ** [String\List[String]]: Result path of string to check if is equal to result value or an array of string or specify an action's input, for example, `input.matchString`. 
```

* **re-execution* ** [String] (force): By default if previous action run is not yet finished, next scheduled run is skipped. If you set `re-execution: 'force'`, the previous run will be killed, stopping the Docker container. 
* **scheduled**:
   * **every* ** [String] format <int\><interval type\>: Time interval between one run and the next one (for example, 10m, 5d, etc.) or specify an action's input i.e. input.every:
     * d: DAYS
     * h: HOURS
     * m: MINUTES

>    * **expire* ** [String] format <int\><interval type\>: Time after the first run to stop scheduling or specify an action's input i.e. input.expire. The last result will be kept :

Reviewer: ```suggestion
   * **expire* ** [String] format <int\><interval type\>: Time after the first run to stop scheduling or specify an action's input, for example, `input.expire`. The last result will be kept :
```

   - path : 'exit_condition'
```

Or using strings array

```
 integration: 'Incident tools'
 name: 'intervallo date loop'
 type: Scheduled
 script:
    code: |
        [......]
 exit_condition:
   - path: 'exit_condition'
     string: 
        - 'Open'
        - 'Pending'
        - 'Waiting'
 re-execution: 'force'  
 scheduled: 
     - every: '10s'
       expire: '120s'
 output:
   - path : 'exit_condition'
```

Or using action's input

> *NOTE: if you use an action's input this input field should be required = true*

Reviewer: ```suggestion
:::note
If you use an action's input, this input field should be `required = true*`.
:::
```

    - path : 'exit_condition'
```

> Other example (using action's input)

Reviewer: ```suggestion
Other example (using action's input):
```
